x-pack/plugins/log_explorer/public/components/virtual_columns/column_tooltips/tooltip_button.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiIcon } from '@elastic/eui';
+import React from 'react';
+import { dynamic } from '../../../utils/dynamic';
+
+const ColumnHeaderTruncateContainer = dynamic(
```

Comment:
**question:** is this component so bundle-heavy that needs lazy loading?

Reply:
This one to be honest i am not sure. But do you see any challenge with Lazy loading ?